socorro/external/postgresql/base.py

```diff
@@ -85,7 +85,7 @@ def query(self, sql, params=None, error_message=None, connection=None):
             if not connection:
                 connection = self.database.connection()
                 fresh_connection = True
-            # self.context.logger.debug(connection.cursor.mogrify(sql, params))
+            # logger.debug(connection.cursor().mogrify(sql, params))
```

Comment:
Yeah me too (and don't let me derail this PR with this discussion!), it honestly wouldn't hurt to have this in prod either and would be helpful sometimes. If we're running with debug on in prod we kind of deserve/expect to have debug output, and this is not so much really.